hypothesize/utilities.py

```diff
@@ -2261,6 +2261,33 @@ def linhat(x, con, est, *args):
 
     return psihat
 
+def mcpKadjp(p,proc="Holm"):
+    
+    """
+    P-value adjustment procedure based on Holms method..
```

Comment:
Need to flesh out.

Reply:
I really appreciate this by the way. I will look things over as soon as I get a spare second at work.

Reply:
No worries. Sorry for not adding the requested functions. Only saw the list after the function had been created for a different project im working on. Will slowly add some from the list.

Reply:
A couple notes having looked things over:

- The results in Wilcox's mcpKadjp package do not seem to match the results in your function. I tried with `p=[0.010 0.030 0.011 0.003]` in both R and Python and got slightly different results.

- To make the output easier to read, the output DataFrame should have labelled columns that match what Wilcox uses. For example,
![scw](https://user-images.githubusercontent.com/4624246/95932215-3c6f0c00-0d99-11eb-8958-0171edf04396.png)

- It would be great to have a doc string that is close to what Wilcox uses in his WRS package (e.g., with the citations). It would also be nice if the doc string matched the format used in other user-facing functions (for example, see [`con1way`](https://github.com/Alcampopiano/hypothesize/blob/56d75ade8cd2341542a770d05ebead3eef45d363/hypothesize/utilities.py#L10) or any of the hypothesis testing functions)

If these issues can be addressed, I would be happy to accept the PR. I would likely even add another correction method so that the function is set up to handle a list of correction methods (as it is in R).